share/python/driver.py

```diff
@@ -442,7 +442,7 @@ def energy(name, **kwargs):
     wfn = procedures['energy'][lowername](lowername, molecule=molecule, **kwargs)
 
     for postcallback in hooks['energy']['post']:
-        postcallback(lowername, **kwargs)
+        postcallback(lowername, wfn=wfn, **kwargs)
 
```

Comment:
The csx4psi is the only use I know of. I don't mind another way but the hooks seems a good way to run anonymous code at certain point in calc. 

Sent from my iPhone

> On May 11, 2016, at 6:40 PM, Daniel Smith <notifications@github.com> wrote:
> 
> In share/python/driver.py:
> 
> > @@ -442,7 +442,7 @@ def energy(name, **kwargs):
> >      wfn = procedures['energy'][lowername](lowername, molecule=molecule, **kwargs)
> >  
> >      for postcallback in hooks['energy']['post']:
> > -        postcallback(lowername, **kwargs)
> > +        postcallback(lowername, wfn=wfn, **kwargs)
> >  
> Do we even use the postcallback feature? I think the more python centric code is taking its place.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>